### PR TITLE
g2 electroshock balance

### PR DESCRIPTION
**Pull request by cuberound**
# About the pull request

reduces stam damage to humans, fixes bracer energy drain

# Explain why it's good for the game
bugfix and the stam dmg was crazy high resaulting in huge stuns and long slow durations on humans

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
balance: reduces g2 electroshock grenade stamina damage
fix: yauta bracers get properly drained by emp
/:cl:

### CODE REVIEW

**Review by Drulikar — CHANGES_REQUESTED** [review]
Other than maybe the changes to `code/modules/cm_preds/yaut_bracers.dm` I don't think any of this is necessary. I don't want the extra availability, I don't want the reduced range, I'd prefer stun over slow.

File: code/game/objects/items/explosives/grenades/marines.dm [hunk 1]
```diff
@@ -605,7 +605,7 @@
 				new /obj/effect/overlay/temp/elec_arc(get_turf(shocked_human))
 				to_chat(mob, SPAN_HIGHDANGER("All of your systems jam up as your main bus is overvolted by [damage_applied*2] volts."))
 				mob.visible_message(SPAN_WARNING("[mob] seizes up from the elctric shock"))
-			shocked_human.take_overall_armored_damage(damage_applied, ARMOR_ENERGY, BURN, 90) // 90% chance to be on additional limbs
+			shocked_human.take_overall_armored_damage(damage_applied, ARMOR_ENERGY, BURN, 100) //100 means the dmg gets spread evenly over the entire body
 			shocked_human.make_dizzy(damage_applied)
 			mob.apply_stamina_damage(damage_applied*human_stam_dam_factor) // Stamina damage
```
[the remark this answers is not in the document]
Comment (Diegoflores31) [reply code/game/objects/items/explosives/grenades/marines.dm:610]: or a weaken

File: code/modules/cm_preds/yaut_bracers.dm [hunk 2]
```diff
@@ -302,7 +302,7 @@
 
 /obj/item/clothing/gloves/yautja/hunter/emp_act(severity)
 	. = ..()
-	charge = max(charge - (severity * 500), 0)
+	charge = max(charge - (1000/severity), 0) //someone made weaker emp have higer severity so we devide
```
Comment (Drulikar) [inline code/modules/cm_preds/yaut_bracers.dm:305]:
```suggestion
	charge = max(charge - (1000/severity), 0) //someone made weaker emp have higer severity so we divide
```

File: code/game/objects/items/explosives/grenades/marines.dm [hunk 3]
```diff
@@ -605,22 +605,22 @@
...
-			mob.Slow(damage_applied/11)
+			mob.Slow(damage_applied/xeno_slowdown_numerator)
 
 		if(mob_dist < 1) // Range based stuff, standing ontop of the equivalent of a canned lighting bolt should mess you up.
 			mob.Superslow(3) // Note that humans will likely be in stamcrit so it's always worse for them when ontop of it and we can just balancing it on xenos.
 			mob.eye_blurry = damage_applied/4
 			mob.Daze(1)
-		else if((mob_dist < (range-1)) && (mob.mob_size < MOB_SIZE_XENO_VERY_SMALL)) // Flicker stun humans that are closer to the grenade and larvas too.
-			mob.apply_effect(1 + (damage_applied/100),WEAKEN) // 1 + damage/40
+		else if((mob_dist < (range-1)) && (mob.mob_size < MOB_SIZE_XENO_VERY_SMALL)) // Flicker slow humans that are closer to the grenade and larvas too.
```
Comment (Drulikar) [inline code/game/objects/items/explosives/grenades/marines.dm:622]:
```suggestion
		else if((mob_dist < (range-1)) && (mob.mob_size < MOB_SIZE_XENO_VERY_SMALL)) // Flicker stun humans that are closer to the grenade and larvas too.
```

File: code/game/objects/items/explosives/grenades/marines.dm [hunk 4]
```diff
@@ -605,22 +605,22 @@
 				new /obj/effect/overlay/temp/elec_arc(get_turf(shocked_human))
 				to_chat(mob, SPAN_HIGHDANGER("All of your systems jam up as your main bus is overvolted by [damage_applied*2] volts."))
 				mob.visible_message(SPAN_WARNING("[mob] seizes up from the elctric shock"))
-			shocked_human.take_overall_armored_damage(damage_applied, ARMOR_ENERGY, BURN, 90) // 90% chance to be on additional limbs
+			shocked_human.take_overall_armored_damage(damage_applied, ARMOR_ENERGY, BURN, 100) //100 means the dmg gets spread evenly over the entire body
```
Comment (Drulikar) [inline code/game/objects/items/explosives/grenades/marines.dm:608]:
```suggestion
			shocked_human.take_overall_armored_damage(damage_applied, ARMOR_ENERGY, BURN, 90) // 90% chance to be on additional limbs
```
I'm fine with the old behavior.

**Review by Drulikar — CHANGES_REQUESTED** [review]

**Review by Drulikar — APPROVED** [review]

**Pull request merged**